scripts/normalize-by-median.py

```diff
@@ -281,6 +284,10 @@ def main():  # pylint: disable=too-many-branches,too-many-statements
         print('loading k-mer counting table from ' + args.loadtable,
               file=sys.stderr)
         htable = khmer.load_counting_hash(args.loadtable)
+        if args.unique_kmers != 0:
+            print('Warning: You have specified a number of unique kmers'
+                  ' but are loading a precreated counting table--'
+                  'argument optimization will NOT be done.', file=sys.stderr)
```

Comment:
Is this what you were concerned about @mr-c? We could change this to

`counting table size will NOT be set automatically.`